dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/web/JakartaDataRecreateServlet.java

```diff
@@ -169,6 +169,70 @@ public void testOLGH28913() throws Exception {
 
         assertEquals(character.getHexadecimal(), result);
     }
+    
+    @Test
+    @Ignore("Reference issue:https://github.com/OpenLiberty/open-liberty/issues/29459")
+    public void testOLGH29459() throws Exception {
+        int x1 = 0, y1 = 0, x2 = 120, y2 = 209;
+        Segment segment = Segment.of(x1, y1, x2, y2);
+        List<Exception> exceptions = new ArrayList<>();
+        
+        if (tx.getStatus() != jakarta.transaction.Status.STATUS_ACTIVE) {
```

Comment:
nitpick: These transaction status checks are not necessary.  In this case, when entering the servlet to execute this test a new/pooled WebServlet thread is used.  The thread context should never start with an active transaction.